lib/Extension/Behat/Behat/Step.php

```diff
@@ -59,8 +60,13 @@ public function path(): string
         return $this->path;
     }
 
-    public function byteOffset(): int
+    public function start(): int
     {
         return $this->startByteOffset;
     }
+
+    public function end(): int
+    {
+        return $this->endByteOffset;
+    }
```

Comment:
this would be better as a `ByteOffsetRange` and a single method `byteOffsetRange()` or similar

Reply:
I used the location object. That made things simpler anyways.